### ISSUE TITLE
[1.3.2.4] published package client code leaks to server

### ISSUE THREAD

**Issue by wojtkowiak** [meteor/meteor#6877]
According to this https://github.com/meteor/meteor/issues/6618 it should be fixed but maybe I am misinterpreting this issue.
I have just published `omega:monitor-widget` using `1.3.2.4` on Windows and on `1.2.1` app and I get
```
C:\projects\app\.meteor\local\build\programs\server\packages\omega_monitor-widget.js:19
W20160421-20:57:30.060(2)? (STDERR) class MonitorWidget {                                                         
W20160421-20:57:30.060(2)? (STDERR) ^^^^^
W20160421-20:57:30.060(2)? (STDERR) SyntaxError: Unexpected reserved word
```

while in package I have:

```javascript
Package.onUse(function monitorWidgetOnUse(api) {
    api.versionsFrom('METEOR@1.2');
    api.use(['templating', 'ecmascript'], 'client');
    api.use('dbernhard:jquery-ui-draggable@0.1.2', 'client');
    api.addFiles([
        'src/monitorWidget.css',
        'src/monitorWidget.html',
        'src/MonitorWidget.js'
    ], 'client');
});
```

Should this be fixed or should I publish using `1.2.1`?
BTW what is the easiest way to publish using old version? Checkout devel at 1.2.1 and publish using it?

**Comment by benjamn** [comment]
The fix for #6618 allowed using Meteor ~1.3 to publish a package for ~1.2 if the package didn't use the `modules` package, but the `modules` package is now implied by the `ecmascript` package, which is why your package is not getting the benefit of the #6618 changes.

That said, it's clear that you're writing a client-only package, and this error is occurring for an unused/lazy file in the server bundle, so this behavior definitely feels like a bug that we should fix. Specifically, we shouldn't be bundling lazy files for a given architecture unless the package uses `modules` on that architecture.

**Comment by benjamn** [comment]
In the meantime, you should be able to select a release when publishing: `meteor publish --release 1.2.1`

**Comment by wojtkowiak** [comment]
Thank @benjamn I did not know `--release` can work in any other case than `meteor update`. Thanks also for planning to fix this.

**Comment by benjamn** [comment]
This will be fixed in Meteor 1.3.3, thanks to https://github.com/meteor/meteor/commit/f6b75305b37d009982033de64ed826266400b3bd!

**Comment by wojtkowiak** [comment]
Thanks a lot! Due to this and https://github.com/meteor/meteor/issues/6843 I am almost feeling like I have a personal support granted :smile:

**Comment by benjamn** [comment]
This should be fixed if you do `meteor update --release 1.3.3-beta.0`.
